pkg/webhook/server.go

```diff
@@ -41,6 +41,9 @@ import (
 // DefaultPort is the default port that the webhook server serves.
 var DefaultPort = 9443
 
+// ExportedTLSConfig is used in unit tests to ensure propagation of tls related configurables to server.
+var ExportedTLSConfig *tls.Config
```

Comment:
Lets find a way to test this without globals

Reply:
Done, hopefully better